ocean-client/src/vault-maxi.ts

```diff
@@ -67,7 +67,7 @@ export async function main(event: maxiEvent, context: any): Promise<Object> {
     }
     console.log('using oceans ' + JSON.stringify(oceansToUse))
 
-    const usedLogId = process.env.VAULTMAXI_LOGID ?? settings.logId
+    const usedLogId = settings.logId
```

Comment:
if we change this here, we also need to add the env var handling to the other stores. otherwise the envVar won't work in AWS anymore

Reply:
I will check this

Reply:
do we even need to change it here? if the envVar is set, its anyway used already (then settings.logId and envVar is the same in the local version. so might just keep the line as it was.